core/src/main/java/org/apache/iceberg/util/PropertyUtil.java

```diff
@@ -100,6 +102,30 @@ public static String propertyAsString(
     return defaultValue;
   }
 
+  /**
+   * Validate the table commit related properties to have non-negative integer on table creation to
+   * prevent commit failure
+   *
+   * @param properties input map
```

Comment:
nit: more descriptive param doc , like key-value map of properties, or just remove this ?

Reply:
I was copying from method below but not really needed this, removed.